Prox/Prox/Data/DatabaseResult.swift

```diff
@@ -4,6 +4,16 @@
 
 import Foundation
 
+// This is an empty class that forms a base class for all places
```

Comment:
Might want to update this to be a little clear (or if we don't want to spend the time, add a brief comment saying this is not entirely accurate but you get the idea ;)).